.cursorrules

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
This file is used to give context to Cursor's AI features on how we want it to generate code. Happy to remove this if you aren't ready to accept this at this time.